docs/source/certificates/certificates.md

```diff
@@ -0,0 +1,173 @@
+## Certificate Management Guide
+
+**Audience**: Fabric network admins
+
+This guide provides a brief overview, resources, and technical details for Fabric network administrators to manage certificates (certs).
+
+### Key Concepts
+
+**Registration** – A registration is a username (enrollment ID) and password stored in the certificate authority (CA). This is created by a CA admin level user and has no expiration. The registration also contains any roles and attributes required.
+
+**Enrollment** – Based on the registration username and password, an enrollment (or identity), can be created. The enrollment contains a public/private key pair and the (encoded) roles and attributes from the registration. The public key in the enrollment is referred to as a certificate, which is a signed certificate that is required to use Hyperledger Fabric.
+
+### Certificate types
+
+Hyperledger Fabric implements two basic types of certificates: 1) **enrollment** (or identity) certificates, and 2) **TLS** certificates.
+
+#### Enrollment Certificates
+
+Enrollment certificates are classed by type into the following four roles:
+
+* **admin**
+* **peer**
+* **orderer**
+* **client**
+
+The roles are used for the following purposes:
+
+<b>admin:</b> Signed certificates used by Hyperledger Fabric to identify identities that are administrative-level user certificates. These identities are required in order to make changes to Hyperledger Fabric configurations.
```

Comment:
Not sure why we add "Signed". All certificates are signed.

Reply:
👍 - strike ~signed~ or replace --> "x509" ?

Reply:
Are all types of enrollment cert x509 or only the admin ones?

Reply:
Are TLS certs x509? We call them "signed" currently.